pkg/actors/actors.go

```diff
@@ -828,6 +828,10 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 }
 
 func (a *actorsRuntime) CreateTimer(ctx context.Context, req *CreateTimerRequest) error {
+	var (
+		err             error
```

Comment:
Could you please add a UT?

Reply:
it means Unit Test?

Reply:
Yes, unit test.

Reply:
[PR 2732](https://github.com/dapr/dapr/pull/2732)